[2025-01-12T16:51:20.237+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: insert_date.md_exchange_rate_d manual__2025-01-12T16:51:14.106276+00:00 [queued]>
[2025-01-12T16:51:20.256+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: insert_date.md_exchange_rate_d manual__2025-01-12T16:51:14.106276+00:00 [queued]>
[2025-01-12T16:51:20.257+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2025-01-12T16:51:20.601+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): md_exchange_rate_d> on 2025-01-12 16:51:14.106276+00:00
[2025-01-12T16:51:20.607+0000] {standard_task_runner.py:60} INFO - Started process 4256 to run task
[2025-01-12T16:51:20.616+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'insert_date', 'md_exchange_rate_d', 'manual__2025-01-12T16:51:14.106276+00:00', '--job-id', '2411', '--raw', '--subdir', 'DAGS_FOLDER/insert_data.py', '--cfg-path', '/tmp/tmpwdyank0o']
[2025-01-12T16:51:20.621+0000] {standard_task_runner.py:88} INFO - Job 2411: Subtask md_exchange_rate_d
[2025-01-12T16:51:20.685+0000] {task_command.py:423} INFO - Running <TaskInstance: insert_date.md_exchange_rate_d manual__2025-01-12T16:51:14.106276+00:00 [running]> on host 711845a6becd
[2025-01-12T16:51:20.788+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='budanovsa' AIRFLOW_CTX_DAG_ID='insert_date' AIRFLOW_CTX_TASK_ID='md_exchange_rate_d' AIRFLOW_CTX_EXECUTION_DATE='2025-01-12T16:51:14.106276+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-12T16:51:14.106276+00:00'
[2025-01-12T16:51:20.819+0000] {base.py:83} INFO - Using connection ID 'postgres-db' for task execution.
[2025-01-12T16:51:20.956+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-01-12T16:51:20.969+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=insert_date, task_id=md_exchange_rate_d, execution_date=20250112T165114, start_date=20250112T165120, end_date=20250112T165120
[2025-01-12T16:51:20.974+0000] {base.py:83} INFO - Using connection ID 'postgres-db' for task execution.
[2025-01-12T16:51:21.036+0000] {sql.py:450} INFO - Running statement: 
        INSERT INTO log.logt (execution_datetime, event_datetime, event_name)
        VALUES ('2025-01-12 16:51:14.106276', '2025-01-12T16:51:20.970556', 'md_exchange_rate_d');
    , parameters: None
[2025-01-12T16:51:21.040+0000] {sql.py:459} INFO - Rows affected: 1
[2025-01-12T16:51:21.108+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-01-12T16:51:21.183+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
